### ISSUE TITLE
[Question] Masked actions PPO in multiagent setting using PettigZoo

### ISSUE THREAD

**Issue by MarcoPicione** [Stable-Baselines-Team/stable-baselines3-contrib#250]
### ❓ Question

Hi, is it possible to train a Parallel PettingZoo environment using MaskablePPO? Just in case how "action_masks" should be implemented?
Thanks :)

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues) in the repo
- [X] I have read the [documentation](https://sb3-contrib.readthedocs.io/en/master/)
- [X] If code there is, it is [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014)
- [X] If code there is, it is formatted using the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by MarcoPicione** [comment]
I have hardcoded inside the NN class the mask. I don't remember the details, but looking in the literature there is a paper that says you can manually put the logits to 0 and in this way everything behaves like a mask. It worked
